docs/explanations/audit-scanner/policy-reports.md

```diff
@@ -7,7 +7,7 @@ title: "Audit Scanner - Policy Reports"
 
 When using the Kubewarden Audit Scanner, the results of the policy scans are
 stored using the
-[PolicyReport](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html)
+[PolicyReport](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/045372e558b896695b2daae92e8c7a04d4d40282/policy-report/docs/index.html)
```

Comment:
pinning the CRD docs to the v1alpha2 that we use, instead of the v1beta incomplete that master shows on their page.